File: original_scripts/c/bubble_sort/deepseek-coder_6.7b.c
Sure, here is the source code for a bubble sort algorithm in C without any comments:

```c
void BubbleSort(int arr[], int n) {
    int i, j;
    for (i = 0; i < n-1; i++)       
       for (j = 0; j < n-i-1; j++)  
           if (arr[j] > arr[j+1]) {
              int temp = arr[j];
              arr[j] = arr[j+1];
              arr[j+1] = temp;
           }
}
```

File: original_scripts/r/bubble_sort/deepseek-coder_6.7b.r
Sure, here is the source code for a Bubble Sort in R without any comments or explanation.

```R
bubble_sort <- function(x) {
    n <- length(x)
    repeat {
        swapped <- FALSE
        for (i in 1:(n - 1)) {
            if (x[i] > x[i + 1]) {
                tmp <- x[i]
                x[i] <- x[i + 1]
                x[i + 1] <- tmp
                swapped <- TRUE
            }
        }
        n <- n - 1
        if (!swapped) return(x)
    }
}
```
This code defines a function `bubble_sort` that takes an input vector `x`. It uses the Bubble Sort algorithm to sort the numbers in this vector. The variable `n` is used to keep track of how many items are left to be checked after each iteration through the loop, and `swapped` is set to TRUE when a swap occurs, indicating that at least one item was out-of-order and needs to be sorted. If no swaps occur during an entire pass through the list, it means the list is already sorted and the function returns it immediately.